### PR TITLE
docker container for thank you mail

### PR DESCRIPTION
**Pull request by Entkenntnis**
open todo: mail configuration

### CODE REVIEW

File: container/thankyou-mail/script.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    auth: {
+      user: "quinn@serlo.org", // generated ethereal user
+      pass: process.env.MAIL_PASSWORD, // generated ethereal password
+    },
+  });
+
+  // send mail with defined transport object
+  let info = await transporter.sendMail({
+    ...mailBase,
+    to: "quinn@serlo.org",
```
Comment (hugotiburtino) [inline container/thankyou-mail/script.js:86]:
```suggestion
    to: $EMAIL_DISTRIBUTOR
```
If you follow the idea of using a distributor (Verteiler) for it.

File: container/thankyou-mail/script.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+"use strict";
+const nodemailer = require("nodemailer");
+const fetch = require("node-fetch");
+
+const users = ["quinn@serlo.org"];
```
Comment (hugotiburtino) [inline container/thankyou-mail/script.js:5]:
```suggestion
```
Unnecessary in case you prefer to use a email distributor

**Review by hugotiburtino — CHANGES_REQUESTED** [review]
Let's prepare it for the real deployment.

File: container/thankyou-mail/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+FROM node:18
```
Comment (kulla) [inline container/thankyou-mail/Dockerfile:1]:
```suggestion
FROM node:18-alpine
```

**Review by kulla — CHANGES_REQUESTED** [review]

**Comment by Entkenntnis** [comment]
TODO:

- [ ] use mailing list
- [ ] add new parameters to dockerfile

**Comment by Entkenntnis** [comment]
outdated

**Pull request closed without merging**